CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v3.13.0dev - [date]
+
+### Enhancements & fixes
+
+- [PR #1049](https://github.com/nf-core/rnaseq/pull/1049) - RSEM imposes own arguments to STAR, therefore `--star_extra_alignment_args` are ignored for this route. A warning is displayed when both are used together.
```

Comment:
```suggestion
- [PR #1049](https://github.com/nf-core/rnaseq/pull/1049) - RSEM imposes own arguments to STAR, therefore `--extra_star_align_args` are ignored for this route. A warning is displayed when both are used together.
```

Reply:
Yikes! How did that happen? 

Yes, there was a meeting interruption in between, and I already had twice as many cups of coffee than usual to survive the day, but screwing up the only variable name in the PR, which I initially I copy & pasted from the modules.config...pathetic.